BlueElectrum.js

```diff
@@ -21,11 +18,11 @@ let wasConnectedAtLeastOnce = false;
 async function connectMain() {
   const host = await AsyncStorage.getItem(AppStorage.ELECTRUM_HOST);
   const tcp = await AsyncStorage.getItem(AppStorage.ELECTRUM_TCP_PORT);
-  const usingPeer = !!host && !!tcp ? { host, tcp } : await getRandomHardcodedPeer();
+  const usingPeer = !!host && !!tcp ? { host, tcp } : defaultPeer;
```

Comment:
I think we should hardcode it and not take from AsyncStorage in the first place, because when user will update the app he can have old ports/hosts stored